### ISSUE TITLE
Icon request: Network icons

### ISSUE THREAD

**Issue by yhojann-cl** [google/material-design-icons#1700]
Please, add new network icons, by example the RJ45 for list network devices like as `lo`, usb and bluethoot hospot or external non wifi network device.

Examples: 

- https://icons.getbootstrap.com/icons/ethernet/
- https://icons.getbootstrap.com/icons/pci-card-network/
- https://icons.getbootstrap.com/icons/router/
- https://icons.getbootstrap.com/icons/modem/

**Comment by sumeidibr** [comment]
I agree with @yhojann-cl ... It would be really good if we had these icons
